package.json

```diff
@@ -92,9 +93,7 @@
     "build-dist": "bash ./scripts/build-dist.sh",
     "build-win-installer": "scripts\\build-windows-installer.bat",
     "check-lockfile": "./scripts/check-lockfile.sh",
-    "lint": "yarn run lint-prettier && eslint . && flow check",
-    "lint-prettier": "node scripts/prettier.js lint",
-    "prettier": "node scripts/prettier.js write",
```

Comment:
Thanks! So it wouldn't be exactly the same since we used to make people run `yarn run prettier` in the script to auto-fix the issues, but I'm assuming someone will run eslint --fix once in a while then.